src/libman/Commands/CleanCommand.cs

```diff
@@ -25,7 +27,11 @@ public CleanCommand(IHostEnvironment hostEnvironment, bool throwOnUnexpectedArg
         {
             Manifest manifest = await GetManifestAsync();
 
-            Task<bool> deleteFileAction(IEnumerable<string> s) => HostInteractions.DeleteFilesAsync(s, CancellationToken.None);
+            Task<bool> deleteFileAction(IEnumerable<string> s)
+            {
+                _totalFilesCleaned +=  s!= null ? s.Count() : 0;
```

Comment:
local variable

Reply:
Also, do we want to be consistent with the other clients and output the number of libraries we deleted (not number of files). We don't list the number of files deleted (we list the files paths but no count for the .vsix).